### ISSUE TITLE
Haxe Roundup 311

### ISSUE THREAD

**Issue by skial** [skial/haxe.io#108]
To add to the roundup todo list, use the following markdown format.
```
- [ ] optional description [link name](/link/to/item) by [Persons Name](link/to/person)
```

**Comment by Tiago-Ling** [comment]
- [ ] Seeking help to find the cause / solve a jittering problem when moving images in different Haxe frameworks [JitterDemo](https://github.com/Tiago-Ling/JitterDemo) by [Tiago Ling](https://twitter.com/@TiagoLing)

**Comment by skial** [comment]
@Tiago-Ling its fine :wink:

**Comment by skial** [comment]
- [ ] If using [OpenFL](https://twitter.com/Open_FL) on Linux checkout the new installer [script](http://openfl.org/builds/haxe). [Direct Download)(http://t.co/mzjNZYrYO9)
- [ ] Atom.io [support](https://atom.io/packages/flow) for snowkit's flow build tool.
- [ ] Origami Crush [beta](https://plus.google.com/u/0/communities/113998599653560379950) for Android available, built with OpenFL via [Romain Lebon](https://twitter.com/2smacks)
- [ ] If you somehow missed it and have any libraries still on google code, [move them](http://thenextweb.com/google/2015/03/12/google-code-is-shutting-down/), its dead.
- [ ] Haxe plugin for IntelliJ IDEA 13 and 14 has been [updated](https://twitter.com/As3Boyan/status/576069141238542337), now available through JetBrains plugin repo.
- [ ] Unity3D ads [extension](https://github.com/ipsilondev/openfl-unityads) for OpenFL by [Ipsilon Developements](https://twitter.com/ipsilondev)
- [ ] Another, [Success Story for Haxe](http://nadako.tumblr.com/post/113390739725/a-success-story-for-haxe) by [Dan Korostelev](https://twitter.com/nadako) - [Hacker News](https://news.ycombinator.com/item?id=9198406)
- [ ] Introduction [post](http://www.accidentalrebel.com/i-created-an-animatedsprite-tool-for-flambe/2015/03/10/) by [Accidental Rebel](https://twitter.com/accidentalrebel) called "I created an AnimatedSprite Tool for Flambe". Cute animations as well.
- [ ] Massive Interactive is [hiring](http://www.cwjobs.co.uk/JobSearch/JobDetails.aspx?JobId=61900084)!
- [ ] WWX2015 [crowdfunding](http://www.ulule.com/wwx2015/) has launched! Currently 36% funded!
- [ ] [Namide](https://twitter.com/_namide) has released [Tetrominos Killer](http://games.namide.com/tetrominos-killer/) created with Haxe which has been freed from Facebook.

**Comment by skial** [comment]
- [ ] [HVM](https://github.com/dpeek/hvm) by [David Peek](https://twitter.com/DavidPeek)

**Comment by skial** [comment]
- [ ] [Lubos Lenco](https://twitter.com/luboslenco) [showing](https://twitter.com/luboslenco/status/576769887714058240) kha plus blender seamlessly importing skeletal animation.
- [ ] [MotionTwin](https://twitter.com/motiontwin) [showing](https://twitter.com/motiontwin/status/576773404478607360) _"the cutest bellboy ever in MonsterHotel"_

**Comment by GildasP** [comment]
I there.
I released the beta version of my game for widows/macosx a few days ago :
http://www.abstractpixelgames.com
It's built with OpenFL / HaxePunk and a home made [HaxePunk port](https://github.com/GildasP/HaxePunk-Processing) of the [Processing](https://processing.org/) vector drawing API.
I would be glad to be in an haxe roundup if you find the game interesting !
I've got a pic you might use :
![capture-gif](https://cloud.githubusercontent.com/assets/713535/6652740/8cd47f4a-ca79-11e4-9c48-b8bafc472896.gif)
and there is a video hosted on the website too.
Don't hesitate to tell me what you think about it.
